resources/views/livewire/public-link-generator.blade.php

```diff
@@ -0,0 +1,53 @@
+<div>
+    @if(!empty($signed_urls))
+        <div class="mb-4 flex flex-col">
+            <p>{{__('Signed Documents')}}:</p>
+            @foreach($signed_urls as $url)
+                <p class="mt-2">{{__($url)}}</p>
+            @endforeach
+        </div>
+    @endif
+    @if(!empty($unsigned_documents))
+        <div class="dropdown-full-w">
+            <x-dropdown width="w-full">
+                <x-slot name="trigger">
+                    <x-button class="w-full" icon="document">
+                        {{ __('Add Signature') }}
+                    </x-button>
+                </x-slot>
+                @foreach($unsigned_documents as $doc)
+                    <x-dropdown.item
+                        wire:click="setPublicLink('{{$doc}}')">
+                        {{ __($doc) }}
+                    </x-dropdown.item>
+                @endforeach
+            </x-dropdown>
+        </div>
+    @endif
+    @if(!empty($generated_urls))
+        <div class="mt-4">
+            @foreach($generated_urls as $label => $link)
+                <x-input
+                    class="mb-2"
+                    label="{{__($label)}}"
+                    readonly
+                    value="{{$link}}"
+                    type="text"
+                    x-ref="link"
```

Comment:
are you sure this works with the same x-ref inside a blade foreach?

Reply:
you are right - it worked last time - now not - i fix